cmd/helper/get_workspace_config.go

```diff
@@ -163,6 +163,19 @@ func findDevcontainerFiles(ctx context.Context, rawSource, tmpDirPath string, ma
 		}
 		log.Debug("Done cloning git repository")
 
+		if gitCommit != "" {
```

Comment:
Is this working? Because when we clone with --depth=1, doesn't that cause any issues?

Reply:
No I checked now, and this isn't working. If we wish to reach to the commit, we will have to either pull the complete repo or play around with the depth using 'git deepen' to reach the commit we want. I feel we can remove the resetting to this commit and just check the head of the clone to verify whether the repository is valid. What do you think?